airbyte-integrations/connectors/source-google-sheets/google_sheets_source/helpers.py

```diff
@@ -152,6 +152,11 @@ def get_sheets_in_spreadsheet(client, spreadsheet_id: str):
         spreadsheet_metadata = Spreadsheet.parse_obj(client.get(spreadsheetId=spreadsheet_id, includeGridData=False))
         return [sheet.properties.title for sheet in spreadsheet_metadata.sheets]
 
+    @staticmethod
+    def get_sheets_properties(client, spreadsheet_id: str):
```

Comment:
should this be called `get_sheet_row_count` ? Also can you add the return value type signature?

Reply:
@sherifnada done